torba/baseaccount.py

```diff
@@ -175,3 +175,38 @@ def get_private_key(self, chain, index):
 
     def get_balance(self):
         return self.ledger.db.get_balance_for_account(self)
+
+
+class SingleKeyAccount(BaseAccount):
+    def __init__(self, ledger, seed, encrypted, private_key,
+                 public_key, receiving_gap=0, change_gap=0,
+                 receiving_maximum_use_per_address=0,
+                 change_maximum_use_per_address=0):
```

Comment:
What are sane default values for `receiving_gap`, `change_gap`, `receiving_maximum_use_per_address`, `change_maximum_use_per_address`?

Reply:
Single Key accounts don't have any of those settings.